infra/k8s/kops-weave/weave.yml

```diff
@@ -172,6 +172,7 @@ items:
             - sh
             - -ac
             - >
+              sleep 20 &&
```

Comment:
`weave` daemonset is currently dependent on the `dummy` daemonset, which is not great - we should not have such dependencies between daemonsets in general, but I don't know how to avoid that one, because the `flannel` interface is lazily created, only when there is a pod on the host.

So for now we add a `sleep 20`, which will delay the `initContainer` on `weave`, hopefully enough so that `dummy` has started - when we start both daemonsets at the same time.

I will add a command as part of the setup so that we can easily confirm that routes on every host are setup correctly during Kubernetes startup.

Reply:
Actually it is not `weave` that is dependent on the DNS route, but rather `sidecar`. So I moved the `initContainer` to `sidecar` and this simplifies things.